docs/visual-basic/developing-apps/programming/app-settings/accessing-application-settings.md

```diff
@@ -52,9 +52,6 @@ This section contains topics describing the `My.Settings` object and the tasks i
 |Save updated user setting values|[How to: Persist User Settings in Visual Basic](../../../../visual-basic/developing-apps/programming/app-settings/how-to-persist-user-settings.md)|  
 |Determine the values of user settings|[How to: Read Application Settings in Visual Basic](../../../../visual-basic/developing-apps/programming/app-settings/how-to-read-application-settings.md)|  
   
-## See Also  
- [Managing Application Settings (.NET)](/visualstudio/ide/managing-application-settings-dotnet)   
```

Comment:
Turns out that I had removed this link then added it back in the previous commit. No further changes, so I'm going to merge.